outlines/models/llamacpp.py

```diff
@@ -83,6 +83,12 @@ def convert_token_to_string(self, token: str) -> str:
         else:
             return token
 
+    def apply_chat_template(
+        self,
+        prompt_or_conversation: Union[str, List[Dict[str, str]]],
+    ) -> str:
+        raise NotImplementedError("Chat templates are not yet supported by llama-cpp")
```

Comment:
Could you either say "not yet supported by Outlines llama-cpp integration" or implement https://github.com/abetlen/llama-cpp-python/blob/master/README.md#chat-completion